lib/jsonapi/view.ex

```diff
@@ -59,6 +59,17 @@ defmodule JSONAPI.View do
   If you always want to include a relationship. First make sure its always preloaded
   and then use the `[user: {UserView, :include}]` syntax in your `includes` function. This tells
   the serializer to *always* include if its loaded.
+
+  There are several options available through configuration. These include switching from underscores
```

Comment:
My thought was if browsing the docs in-terminal the README wouldnt be there if you're doing `h JSONAPI.View`, and this makes sure its available to both casual browsers on hex.pm/github and also those who use the IEx style of doc-reading.